Extension/src/Utility/Process/commandLine.ts

```diff
@@ -0,0 +1,245 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { spawnSync } from 'child_process';
+import { OperatingSystem } from '../../constants';
+import { resolveEnvironmentVariables } from '../System/environment';
+
+function windows(argsString: string): string[] {
```

Comment:
Is this utility intended to replace word-exp?  cc: @colengms

Reply:
Yes; I've been talking to @Colengms  about this -- this will let us drop the native binary. I haven't replaced the actual usage of this at this point yet.